src/app/Helpers/Stats.php

```diff
@@ -23,6 +23,65 @@ public static function countFutureAvailability() {
       return Location::available()->count();
     }
 
+    /**
+     * Get the number of providers that have never had their data
+     * updated.
+     */
+    public static function countNeverUpdated() {
+        return Location::whereNull('updated_at')->count();
```

Comment:
This one will probably need to be more a little complex. The `updated_at` column will get changed any time we make changes to the Location even if we aren't updating Availability. I think we need another relationship function to get availabilities including ones in the past that have been deleted. I'd suggest the below:

```
public function alltimeAvailabilities() {
    return $this->hasMany('App\Models\Availability', 'location_id')->withTrashed();
}

public static function countNeverUpdated() {
        return Location::has('alltimeAvailabilities', '<', 1)
}```